tox.ini

```diff
@@ -0,0 +1,35 @@
+[tox]
+requires =
+    tox-conda
+
+[testenv]
+conda_spec =
+    py38: {toxinidir}{/}requirements{/}locks{/}py38-linux-64.lock
+    py39: {toxinidir}{/}requirements{/}locks{/}py39-linux-64.lock
+    py310: {toxinidir}{/}requirements{/}locks{/}py310-linux-64.lock
+usedevelop =
+    true
+
+[testenv:py{38,39,310}-tests]
+description = Perform Iris unit, integration and system tests.
+commands =
+    pytest -n auto {posargs} {toxinidir}{/}lib{/}iris{/}tests
+
+[testenv:py{38,39,310}-docs{,-linkcheck,-tests}]
+description = Build, and optionally linkcheck or test, the Iris documentation
+allowlist_externals =
+    make
+changedir =
+    {toxinidir}{/}docs
+commands =
+    make clean html
+    linkcheck: make linkcheck
+    tests: make doctest
+
+[testenv:py{38,39,310}-wheel]
+description = Install wheel and test the Iris import
+skip_install = true
+deps =
+    {toxinidir}{/}dist{/}scitools_iris*.whl
```

Comment:
Note that with [nox](https://github.com/SciTools/iris/blob/main/noxfile.py#L288-L294) we caught the scenarios where there were no wheels and where there were more than one wheel. I could have added this checking for tox, I would have had to remove this `deps` section and instead set the commands to ensure it gets one wheel , then to manually pip install it.

But I felt this was more simple and the above was unnecessary complication.
I tested to see how tox would handle the wrong number of wheels:

1) Two wheels available (iris3.5 and iris3.7):
```
$ tox -e py310-wheel
py310-wheel create: .../iris/.tox/py310-wheel
py310-wheel installcondadeps: --file=.../iris/requirements/locks/py310-linux-64.lock
WARNING: could not determine version of: .../iris/dist/scitools_iris-3.5.0.dev51+dirty-py3-none-any.whl
WARNING: could not determine version of: .../iris/dist/scitools_iris-3.7.0.dev51+dirty-py3-none-any.whl
_______________________________________________________________ summary ________________________________________________________________
  py310-wheel: commands succeeded
  congratulations :)
Traceback (most recent call last):
...
tox.exception.MissingDependency: MissingDependency:.../iris/dist/scitools_iris*.whl
```

2) no wheels available
```
$ tox -e py310-wheel
py310-wheel create: .../iris/.tox/py310-wheel
py310-wheel installcondadeps: --file=.../iris/requirements/locks/py310-linux-64.lock
_______________________________________________________________ summary ________________________________________________________________
  py310-wheel: commands succeeded
  congratulations :)
Traceback (most recent call last):
...
tox.exception.MissingDependency: MissingDependency: .../iris/dist/scitools_iris*.whl
```

So my preferrence is to just rely on tox's error message

Reply:
Regarding the `WARNING: could not determine version of` in the scenario of two wheels present, those warnings don't appear when there is only one wheel:
```
]$ tox -e py310-wheel
py310-wheel create: .../iris/.tox/py310-wheel
py310-wheel installcondadeps: --file=.../iris/requirements/locks/py310-linux-64.lock
py310-wheel installdeps: .../iris/dist/scitools_iris-3.5.0.dev51+dirty-py3-none-any.whl
...
py310-wheel run-test-pre: PYTHONHASHSEED='1378407663'
py310-wheel run-test: commands[0] | python -c 'import iris; print(f'"'"'{iris.__version__=}'"'"')'
iris.__version__='3.5.0.dev51+dirty'
_______________________________________________________________ summary ________________________________________________________________
  py310-wheel: commands succeeded
  congratulations :)

```

Reply:
I'm not especially worried about this. I expect Bill only originally included the defensive code because things like that are easier with Nox. So I'm comfortable with your proposal.